modules/nw-aws-nlb-new-cluster.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic--aws-network-load-balancer.adoc
+
+[id="nw-aws-nlb-new-cluster_{context}"]
+= Configuring an Ingress Controller Network Load Balancer on a new AWS cluster
+
+You can create an ingress controller backed by an AWS Network Load Balancer (NLB) on a new cluster.
+
+.Prerequisites
+
+* You must have an installed AWS cluster with network customizations.
+* You must have completed the Creating the installation configuration file procedure.
+* `PlatformStatus` of the infrastructure resource must be AWS.
+** To verify that the `PlatformStatus` is AWS, run:
++
+[source,terminal]
+----
+$ oc get infrastructure/cluster -o jsonpath='{.status.platformStatus.type}'
+AWS
+----
+
+.Procedure
```

Comment:
actually to install NLB on a new AWS cluster, the `Prerequisites` and `Procedure` is very similar to the [section](https://docs.openshift.com/container-platform/4.5/installing/installing_aws/installing-aws-network-customizations.html#modifying-nwoperator-config-startup_installing-aws-network-customizations).
That means:
1. Prerequisites is just `Create the install-config.yaml file and complete any modifications to it`. Since the cluster is not ready yet, so it is impossible to run any `oc get` command to verify that the `PlatformStatus` is AWS.
2. For procedure, the first step is same 
```
Use the following command to create manifests: 
$ ./openshift-install create manifests --dir=<installation_directory>
```
3. the second step is already listed here: `Create a file named `cluster-ingress-default-ingresscontroller.yaml` with the following contents:`

Reply:
My idea is moving the content of `Configuring an Ingress Controller Network Load Balancer on a new AWS cluster` to and follow the [section](https://docs.openshift.com/container-platform/4.5/installing/installing_aws/installing-aws-network-customizations.html#modifying-nwoperator-config-startup_installing-aws-network-customizations), since the custom file named `cluster-ingress-default-ingresscontroller.yaml` should be created manually before the real [`Deploying the cluster`](https://docs.openshift.com/container-platform/4.5/installing/installing_aws/installing-aws-network-customizations.html#installation-launching-installer_installing-aws-network-customizations).

Reply:
Thank you, @lihongan! I can build the file in both places just to cover our bases. I'll push that update in a few minutes to see how it looks.

Reply:
@lihongan I went through and updated the procedure for this one to make the flow easier to follow for the user.

Reply:
@bmcelvee thank you. It looks good to me.